Maple2.Server.Game/Packets/CinematicPacket.cs

```diff
@@ -149,12 +149,16 @@ public static ByteWriter RemoveBalloonTalk(int objectId) {
     }
 
     public static ByteWriter Caption(string type, string title, string script, Align align, float offsetRateX, float offsetRateY, int duration, float scale) {
+        // Convert enum align value to camelCase string
```

Comment:
why don't we just do this on the enum instead? if it has to be centerLeft, centerRight, why not just rename the enums?